### DISCUSSION TITLE
Award

### DISCUSSION THREAD

**Discussion by 2nutz4u** [Security-Onion-Solutions/securityonion#8897] [Show and tell] [answered]
Firstly, sorry for putting it here, but I know the team members monitor the discussion board and I would like to thank you for the wonderful product. We here at the Village of Carol Stream have won an award for your product, which we summited to IMRA. This is a prestigious award. This award was highlighted at the board of directors meeting, which included many local government agencies throughout Illinois. To that point or follow up to that, I would like one of your member to present your product at one of our quarterly GMIS meeting (https://www.gmisillinois.org/)  if possible. Thank you again.

**Comment by phil1090** [comment, marked as answer]
Thank you for the note! I have responded privately to your website submission.
Phil Plantamura
